config.default.json

```diff
@@ -282,18 +283,18 @@
           "auth": {
             "type": "header",
             "name": "Authorization",
-            "value": "Bearer $ACCESS_TOKEN"
+            "value": "Bearer $GHSA_ACCESS_TOKEN"
```

Comment:
Not so critical, but since this is now used both for GitHub Cloud and Server apps, this perhaps could be changed to GHA_ACCESS_TOKEN?

Reply:
yes, I thought about that but didn't want to break the existing setup. But you're right, it's the right time to do it. Amending PR now.